Bottom: 6ee88c4f0f59814eb1dadf471081d1c5b98814aa
Top:    def6df29c5b0b8e696379ed1d546ecd5a4e23651
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-15 18:39:38 +0000

find.h: Annotate which exported functions are not used!


---

diff --git a/src/find.h b/src/find.h
index 4c9279d..60a5d93 100644
--- a/src/find.h
+++ b/src/find.h
@@ -47,7 +47,7 @@ bool LineLineIntersect (LineType *, LineType *);
 bool LineArcIntersect (LineType *, ArcType *);
 bool PinLineIntersect (PinType *, LineType *);
 bool LinePadIntersect (LineType *, PadType *);
-bool ArcPadIntersect (ArcType *, PadType *);
+/* NOT USED EXTERNALLY */// bool ArcPadIntersect (ArcType *, PadType *);
 void LookupElementConnections (ElementType *, FILE *);
 void LookupConnectionsToAllElements (FILE *);
 void LookupConnection (Coord, Coord, bool, Coord, int, bool AndRats);
@@ -56,11 +56,11 @@ bool ResetFoundLinesAndPolygons (bool, int flag);
 bool ResetFoundPinsViasAndPads (bool, int flag);
 bool ResetConnections (bool, int flag);
 void InitConnectionLookup (void);
-void InitComponentLookup (void);
-void InitLayoutLookup (void);
+/* NOT USED EXTERNALLY */// void InitComponentLookup (void);
+/* NOT USED EXTERNALLY */// void InitLayoutLookup (void);
 void FreeConnectionLookupMemory (void);
-void FreeComponentLookupMemory (void);
-void FreeLayoutLookupMemory (void);
+/* NOT USED EXTERNALLY */// void FreeComponentLookupMemory (void);
+/* NOT USED EXTERNALLY */// void FreeLayoutLookupMemory (void);
 void RatFindHook (int, void *, void *, void *, bool, bool);
 void SaveFindFlag (int);
 void RestoreFindFlag (void);
